app/controllers/users/passwords_controller.rb

```diff
@@ -1,34 +1,36 @@
 # frozen_string_literal: true
 
-class Users::PasswordsController < Devise::PasswordsController
-  # GET /resource/password/new
-  # def new
-  #   super
-  # end
+module Users
+  class PasswordsController < Devise::PasswordsController
+    # GET /resource/password/new
+    # def new
+    #   super
+    # end
 
-  # POST /resource/password
-  # def create
-  #   super
-  # end
+    # POST /resource/password
+    # def create
+    #   super
+    # end
 
-  # GET /resource/password/edit?reset_password_token=abcdef
-  # def edit
-  #   super
-  # end
+    # GET /resource/password/edit?reset_password_token=abcdef
+    # def edit
+    #   super
+    # end
 
-  # PUT /resource/password
-  # def update
-  #   super
-  # end
+    # PUT /resource/password
+    # def update
+    #   super
+    # end
 
-  # protected
+    # protected
 
-  # def after_resetting_password_path_for(resource)
-  #   super(resource)
-  # end
+    # def after_resetting_password_path_for(resource)
+    #   super(resource)
+    # end
 
-  # The path used after sending reset password instructions
-  # def after_sending_reset_password_instructions_path_for(resource_name)
-  #   super(resource_name)
-  # end
+    # The path used after sending reset password instructions
+    # def after_sending_reset_password_instructions_path_for(resource_name)
+    #   super(resource_name)
+    # end
+  end
 end
```

Comment:
I'm cleaning this devise code in my PR, worry not!

Reply:
Thanks!